trace/trace.go

```diff
@@ -471,22 +471,27 @@ func (s *Span) send() {
 	s.childrenLock.Lock()
 	// classify span type
 	var spanType string
+	var spanKind string
 	switch {
 	case s.isRoot:
 		if s.parentID == "" {
 			spanType = "root"
+			spanKind = "SERVER"
```

Comment:
Is this definitely going to correctly identify server/client/internal? Sounds like we've had [issues with this code](https://honeycomb.quip.com/eCG0AzChfSe0#RDRADAoRZaO) correctly identifying "spanType". 

OTEL defers to instrumentation to identify [server](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/fe426e42f5fd35422b91d463c0624cd06f0e6330/instrumentation/github.com/gorilla/mux/otelmux/mux.go#L138)/[client](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/2bb24f926b80e0d13e278cbeed85f4d10ffbccb3/instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go#L212) - should we use the same pattern?

Reply:
Forgot we decided to update this in a separate PR

Reply:
I removed from this secstion, and started with an update for the gorilla wrapper but can drop it out in next commit.